packages/diagrams/frontend/sirius-components-diagrams/src/renderer/palette/group-tool/GroupPalette.tsx

```diff
@@ -43,73 +58,229 @@ const usePaletteStyle = makeStyles((theme) => ({
     gridAutoRows: '28px',
     placeItems: 'center',
   },
-  toolIcon: {
-    color: theme.palette.text.primary,
+  toolSection: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginLeft: '8px',
```

Comment:
Could you use theme spacing instead? For other properties also if it is possible?

Reply:
done